Utah.xcodeproj/xcshareddata/xcschemes/Utah.xcscheme

```diff
@@ -71,7 +71,7 @@
          <BuildableReference
             BuildableIdentifier = "primary"
             BlueprintIdentifier = "653A254C283387FE005D4D48"
-            BuildableName = "U-Step.app"
+            BuildableName = "Utah.app"
```

Comment:
Not sure why this changed?

Reply:
Hmm Im not sure either, shoot. Were you able to resolve this?

Reply:
Nah, still working on it